src/routes/reference/secondary-primitives/create-render-effect.mdx

```diff
@@ -16,56 +16,148 @@ description: >-
   effects as DOM creates, before refs are set or connected.
 ---
 
+The `createRenderEffect` primitive creates a reactive computation.
+It automatically tracks reactive values, such as signals, that are accessed within the provided function.
+This function re-runs whenever any of its dependencies change.
+
+Unlike [`createEffect`](/reference/basic-reactivity/create-effect), which runs after the rendering phase is complete, `createRenderEffect` runs immediately as the component is being rendered.
+
+## Execution Timing
+
+- The initial run of a render effect happens during the current rendering phase.
+- It runs as DOM elements are being created and updated, but before they are mounted.
+  As a result, refs are _not_ set before a render effect runs for the first time.
+- If multiple dependencies are updated within the same batch, the render effect only runs once.
```

Comment:
> If multiple dependencies are updated within the same batch, the render effect only runs once.

Id say (or something similar)

> If multiple dependencies are updated within the same batch, the render effect will run one more time.

Because, when dependencies changed, the render effect already ran. So it has to run again, but yes, even if multiple dependencies has been updated, it will run at the very minimum 1 more time.